.github/workflows/build.yml

```diff
@@ -11,7 +11,7 @@ on:
 env:
   REMOVE_BUNDLED_BOOST : rm -rf /usr/local/share/boost
   BUILD_DEFAULT_LINUX: |
-        cmake -S . -B build -D ARCH="default" -D BUILD_TESTS=ON -D CMAKE_BUILD_TYPE=Release && cmake --build build -j3
+        cmake -S . -B build -D ARCH="default" -D BUILD_TESTS=ON -D CMAKE_BUILD_TYPE=Release && cmake --build build --parallel 4
```

Comment:
The rules here are a bit more complicated because there's also a memory requirement: see https://github.com/monero-project/monero/pull/9510#issuecomment-2405956640 (and subsequent discussion). I have a commit ready to PR which accounts for all the requirements, but I'm waiting for this one (and the corresponding main branch PR) to get merged.